_docs-sources/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual_steps.md

```diff
@@ -0,0 +1,122 @@
+
+Configure a hardware MFA device, as suggested by recommendation 1.6. We suggest using a
+[Yubikey](https://www.yubico.com/) due to its strong security characteristics and multitude of form
+factors. Refer to
+[the documentation for more information on using a hardware device with the root user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_physical.html#enable-hw-mfa-for-root).
+
+## 3. Answer security questions and complete contact details
+
+When setting up a new account, AWS asks for contact information and security questions. Unfortunately, there is no API or automation available for this functionality. In the AWS console, visit the [Account settings](https://console.aws.amazon.com/billing/home?#/account) page and complete the _Alternate Contacts_ and _Configure Security Challenge Questions_ questions.
+
+
+## 4. Delete default VPCs and rules from default security groups
+
+Use [cloud-nuke](https://github.com/gruntwork-io/cloud-nuke) to remove the rules from the default VPC and the default ingress/egress rules from the default security groups. Note that it isn’t possible to actually delete the default security group, so instead the command deletes the rules, eliminating the risk of something being mistakenly exposed.
+
+Authenticate on each account (shared, security, dev, prod etc), and then run the following command:
+
+Usage:
+```bash
+cloud-nuke defaults-aws
+```
+
+Example:
+```bash
+aws-vault exec dev -- cloud-nuke defaults-aws
+```
+
+## 5. Subscribe to `BenchmarkAlarmTopic` Alarms
+
+The Config alerts and CloudWatch Metric Alarms all go to the SNS topic `BenchmarkAlarmTopic`. Unfortunately, there is no way to automate
+subscribing to the SNS topic as each of the steps require validating the delivery target. For each deployed account, follow the steps outlined in
+the [AWS docs](https://docs.aws.amazon.com/sns/latest/dg/sns-user-notifications.html) to be notified by Email, Phone,
+or SMS for each of the alerts.
+
+You can also configure an automated system integration if you have a third party alerting system or central dashboard.
+Follow the steps in the [AWS
+docs](https://docs.aws.amazon.com/sns/latest/dg/sns-http-https-endpoint-as-subscriber.html) on how to add an HTTPS endpoint as a subscriber to the alerts.
+
+## 6. Manually maintain buckets to analyze in the `buckets_to_analyze` variable
+
+To set up Macie to analyze the desired S3 buckets, you’ll need to create a **Macie classification job**. Typically,
+you’ll want it to analyze all the buckets in the region. However, the terraform AWS provider does not support specifying
+all the buckets in a region - it requires that an explicit list of buckets be provided. Therefore, you’ll
+need to maintain an explicit list of buckets per region, namely in the variable `buckets_to_analyze`. Please read the
+[documentation](https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/modules/security/macie/variables.tf#L21-L30)
```

Comment:
1. How should customers know what S3 buckets are in the Ref Arch already? We should give them some way to find out. Ideally, of course, we'd pre-configure the Ref Arch with this list out-of-the-box, but that could be a separate PR some time in the future.
2. We should mention customers need to maintain this list in the future as they add new buckets too.

Reply:
> Ideally, of course, we'd pre-configure the Ref Arch with this list out-of-the-box

We do configure with some buckets, but this list is incomplete. There is a task to fix on this sprint.

Reply:
Updated the docs with the AWS CLI commands to retrieve the bucket's list... I wonder now if we could instead give the customer's an script that already returns an list that's ready to be pasted in the terragrunt.hcl. 

I will add this as a TODO and we can improve later.